static/js/containers/App.js

```diff
@@ -177,18 +178,30 @@ class App extends React.Component<Props> {
       return <Redirect to={AUTH_REQUIRED_URL} />
     }
 
+    const isCourseUrl = pathname.startsWith(`/courses`)
+
     return (
       <div className="app">
         <MetaTags>
           <title>MIT Open Learning</title>
         </MetaTags>
-        <Toolbar
-          toggleShowDrawer={this.toggleShowDrawer}
-          toggleShowUserMenu={this.toggleShowUserMenu}
-          showUserMenu={showUserMenu}
-          profile={profile}
-        />
-        <Drawer />
+        {isCourseUrl ? (
+          <CourseToolbar
+            toggleShowUserMenu={this.toggleShowUserMenu}
+            showUserMenu={showUserMenu}
+            profile={profile}
+          />
+        ) : (
+          <React.Fragment>
+            <Toolbar
+              toggleShowDrawer={this.toggleShowDrawer}
+              toggleShowUserMenu={this.toggleShowUserMenu}
+              showUserMenu={showUserMenu}
+              profile={profile}
```

Comment:
why not just pass the `isCourseUrl` boolean into `Toolbar` and conditionally render things in there? I think you'll end up writing less code that way